File: nathan/Problema0444/problema0444.js
const prompt = require("prompt-sync")({ sigint: true });
function verificaN(x){
    if(x < 0)
        console.log("NEGATIVO");
    else
        console.log("POSITIVO");
}
x = Number(prompt("Digite um numero para saber se é positivo ou negativo:"));
verificaN(x);

File: nathan/Problema0444/problema0444.java
package Problema0444;

import java.util.Scanner;

public class problema0444 {

    static void verificaN(int x){
        if(x < 0)
            System.out.println("NEGATIVO");
        else
            System.out.println("POSITIVO");
    }
    public static void main(String[] args) {      
        Scanner input = new Scanner(System.in);

        System.out.println("Digite um numero para saber se é positivo ou negativo: ");
        int n = Integer.valueOf(input.nextLine());
        verificaN(n);
    }
}
